test/vsp/testBsm/test_hostBsm.py

```diff
@@ -1,3 +1,5 @@
+ #!/usr/bin/env python3

```

Comment:
Did you add this shebang on purpose? This should allow you to just call this as a script.
```
./test_hostBsm.py
```

Reply:
I used  this to make sure the script is running by python3. I can remove this if we running with a script.

Reply:
I think this article maybe helpful (https://realpython.com/python-shebang/). Specifically it states "In short, a [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) is a special kind of [comment](https://realpython.com/python-comments-guide/) that you may include in your source code to tell the operating system’s [shell](https://en.wikipedia.org/wiki/Shell_(computing)) where to find the interpreter for the rest of the file"
"A shebang is only relevant to [runnable scripts](https://realpython.com/run-python-scripts/) that you wish to execute without explicitly specifying the program to run them through."
Basically having the shebang and then calling the script `python3 <script_name>` defeats the purpose of the shebang.